### PR TITLE
Add VMware as partner and link it to logo on homepage

### PR DESCRIPTION
**Pull request by Santosh1176**
Signed-off-by: Santosh Kaluskar <dtshbl@gmail.com>

**Description**
Adds VMware as a partner and link it to the logo on the homepage

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
@Santosh1176 we've got to do something about your linter/formatter. The overwhelming majority of the changes here are not necessarily desired.

File: src/sections/Home/Partners-home/partners-home-data.js [hunk 1]
```diff
@@ -59,10 +59,16 @@ export const partners = [
...
-    innerDivStyle:"partner__block__inner"
-    
+    imageRoute: "",
+    innerDivStyle: "partner__block__inner"
+
+  }, {
+    name: "VMware",
+    imageLink: vmware,
+    imageRoute: "/partners#VMware",
+    innerDivStyle: "partner__block__inner",
```
Comment (Nikhil-Ladha) [inline src/sections/Home/Partners-home/partners-home-data.js:71]: We already have VMware added just above this, no need to add it again.
Just update the `imageRoute` above to point to `/partners#VMware`

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit 04d6db2df8ab586ea367fdcac83e994dd115e80a at: https://62bd6bd4c7cb036a196dbec8--layer5.netlify.app

**Comment by leecalcote** [comment]
Merge conflicts, @Santosh1176

**Comment by l5io** [comment]
🚀 Preview for commit 2285f3acde2db605b585c62368a964f9968d4c6d at: https://62bfe2c2207c5202a6aaae79--layer5.netlify.app

**Review by leecalcote — APPROVED** [review]
LGTM

**Review by Nikhil-Ladha — APPROVED** [review]

**Comment by Santosh1176** [comment]
Thank you!

**Pull request merged**